### PR TITLE
Server side comment changes: Replies, parents

### PR DESCRIPTION
**Pull request by andrewconner**
Sorry for the nasty diffs. I made it so blank lines do not have spacing.

### CODE REVIEW

**Comment by andrewconner** [comment]
@eishay Mind reviewing and merging? This is the minimum the frontend needs right now.

File: shoebox/app/com/keepit/controllers/UserController.scala [hunk 1]
```diff
@@ -71,7 +72,15 @@ object UserController extends FortyTwoController {
 
     Ok(userWithSocialSerializer.writes(socialUsers)).as(ContentTypes.JSON)
   }
+  
+  def getSocialConnections() = AuthenticatedJsonAction { authRequest =>
+    val socialConnections = CX.withConnection { implicit c =>
+      SocialConnection.getFortyTwoUserConnections(authRequest.userId).map(uid => User.get(uid)).map(UserWithSocial.toUserWithSocial).toSeq
```
Comment (eishay) [inline shoebox/app/com/keepit/controllers/UserController.scala:78]: look at what we have at UserWithSocial, we should rename it to UserWithSocialForAdmin or something. you don't need all that data there and its very expensive to load it all from the db.

**Comment by andrewconner** [comment]
@eishay Mind reviewing and merging?

File: chrome/keep.js [hunk 2]
```diff
@@ -681,13 +681,11 @@ function startHandShake(callback){
     "json"
   ).error(function(error) {
     log(error.responseText);
-    alert("got error resposne" + error.responseText);
+    //alert("got error resposne" + error.responseText);
```
Comment (eishay) [inline chrome/keep.js:684]: remove this line?

**Pull request merged**